### ISSUE TITLE
Automatically use the currentTask, if set.

### ISSUE THREAD

**Issue by bsaiken** [FWAJL/FieldWorkManager#1061]
When working on a Task ($currentTask is set), adding field data, lab data, service provider - these are automatically added to not just the Project/PM, but also the Task.

**Comment by WebDevJL** [comment]
@bsaiken 

So the main underlying functionality that you want when editing a task and:
- adding a technician, is that we add a row in the tables
  - technician
  - task_technician
- adding a location, is that we add a row in the tables
  - location, 
  - project_location 
  - task_location
- adding a Field Parameter, is that we add a row in the tables
  - field_analyte
  - task_field_analyte
- adding a Lab Analyte, is that we add a row in the tables
  - lab_analyte
  - task_lab_analyte
- adding a service, is that we add a row in the tables
  - service, 
  - project_service 
  - task_service

Is that correct?

**Comment by bsaiken** [comment]
@WebDevJL 
Yes, I thought this was obvious, so I did not list each one.  No harm in clarifying, though.

**Comment by WebDevJL** [comment]
@bsaiken Done for the technicians. Code committed to feature_PM if you want to test.

**Comment by bsaiken** [comment]
@WebDevJL 
Did you mean to assign me to this and change the label to function review?

**Comment by WebDevJL** [comment]
@bsaiken No. I'm just letting you know that I have done it for the technicians and today for the locations. The rest will come soon.

You can test on feature_PM branch if you need.

**Comment by bsaiken** [comment]
@WebDevJL 
I did test on the feature_PM branch and it did not work for technicians.

**Comment by WebDevJL** [comment]
@bsaiken

Please post a screenshot of chrome dev console with the response after
clicking add on the page technician/showForm.

Also do another pull and retest please.

Thank you.

**Comment by bsaiken** [comment]
@WebDevJL 
I would do that, of course, but there was not response.

**Comment by FWAJL** [comment]
@bsaiken 

It works on that branch. See my screenshots:
![screen shot 2015-08-13 at 19 26 19](https://cloud.githubusercontent.com/assets/7524388/9256710/60465e54-41f1-11e5-8991-404b86485e10.png)
![screen shot 2015-08-13 at 19 26 33](https://cloud.githubusercontent.com/assets/7524388/9256713/604efb68-41f1-11e5-9e31-1772e75cdcee.png)
![screen shot 2015-08-13 at 19 26 51](https://cloud.githubusercontent.com/assets/7524388/9256711/604d750e-41f1-11e5-8f20-12882c637bd1.png)
![screen shot 2015-08-13 at 19 26 59](https://cloud.githubusercontent.com/assets/7524388/9256712/604de3a4-41f1-11e5-84b2-12028abe5263.png)

If you really can't get it to work, check in the dev tools the tab "Elements" and check if you see a HTML comment that looks like an error logged. It is always at the very top (see rectangle in red below).
![screen shot 2015-08-13 at 19 30 27](https://cloud.githubusercontent.com/assets/7524388/9256815/28883b4e-41f2-11e5-87db-30d904312c28.png)

**Comment by bsaiken** [comment]
@FWAJL 
Just push it to dev
